elide-core/src/main/java/com/yahoo/elide/parsers/JsonApiParser.java

```diff
@@ -25,7 +26,7 @@
  */
 public class JsonApiParser {
 
-    private final static Pattern DUPLICATE_SEPARATOR_PATTERN = Pattern.compile("/+");
+    private final static Pattern DUPLICATE_SEPARATOR_PATTERN = Pattern.compile("//+");
```

Comment:
We want to replace 2 or more with a single slash.  No need to waste time matching single slashes.